components/layout_2020/lib.rs

```diff
@@ -33,7 +34,7 @@ use style::values::computed::{Length, LengthOrAuto};
 use crate::geom::LogicalVec2;
 
 pub struct ContainingBlock<'a> {
-    inline_size: Length,
+    inline_size: Au,
     block_size: LengthOrAuto,
```

Comment:
Seems a bit strange to have an Au in one axis, but a LengthOrAuto in the other.
Would it add much complexity to simultaneously change block_size into AuOrAuto?

Reply:
Earlier, I wasn't sure about how we wanted to proceed with the usage of `LengthOrAuto`, `LengthPercentage`, and `LengthPercentageOrAuto` with `Au`. That's why I didn't push those changes in this PR, but I do have a [wip branch](https://github.com/atbrakhi/servo/tree/make_block_size_use_au) that is making `block_size` use `AuOrAuto`. It still needs some work.

If you prefer, I can push those changes in this PR and we can do a wpt run, or I can open a separate PR for that?

Reply:
I think it's a bit better to do them together, but not a big deal if you prefer to do them separately.

Reply:
I had finished those changes already so pushed it to this PR. Let see how wpt feels about these changes!